### ISSUE TITLE
Can't update cells home to v3

### ISSUE THREAD

**Issue by sergeycherepanov** [pydio/cells#374]
When I replaced the binary it doesn't want to run. Where I can find the guide how to migrate from v2 to v3?

```
panic: assignment to entry in nil map
goroutine 1 [running]:
github.com/pydio/cells/common/config/migrations.updateVersionsStore(0x75f2680, 0xc000ba2ae0, 0xc000b7d700, 0xc000b82e76)
        github.com/pydio/cells/common/config/migrations/2.2.99.go:63 +0x46f
github.com/pydio/cells/common/config/migrations.getMigration.func1.1(0x75be620, 0xc00011c000, 0xffffffffffffffff, 0xc0000b3900)
        github.com/pydio/cells/common/config/migrations/migrations.go:52 +0x3a
github.com/pydio/cells/common/utils/migrations.Apply(0x75be620, 0xc00011c000, 0xc0000b3880, 0xc0000b33c0, 0xc000af0b00, 0x9, 0x10
        github.com/pydio/cells/common/utils/migrations/migrations.go:56 +0xd9
github.com/pydio/cells/common/config/migrations.UpgradeConfigsIfRequired(0x75f2680, 0xc000ba2ae0, 0xc0000b33c0, 0x0, 0x0, 0xc000b
        github.com/pydio/cells/common/config/migrations/migrations.go:80 +0x355
github.com/pydio/cells/cmd.initConfig(0xc000638c60)
        github.com/pydio/cells/cmd/root.go:317 +0x8a5
github.com/pydio/cells/cmd.glob..func81(0x8d01aa0, 0x92f3480, 0x0, 0x0, 0x0, 0x0)
        github.com/pydio/cells/cmd/start.go:150 +0x8ce
github.com/pydio/cells/vendor/github.com/spf13/cobra.(*Command).execute(0x8d01aa0, 0x92f3480, 0x0, 0x0, 0x8d01aa0, 0x92f3480)
        github.com/pydio/cells/vendor/github.com/spf13/cobra/command.go:839 +0x530
github.com/pydio/cells/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x8d01d40, 0xc00011c000, 0x75be5e0, 0xc000730580)
        github.com/pydio/cells/vendor/github.com/spf13/cobra/command.go:958 +0x375
github.com/pydio/cells/vendor/github.com/spf13/cobra.(*Command).Execute(...)
        github.com/pydio/cells/vendor/github.com/spf13/cobra/command.go:895
github.com/pydio/cells/vendor/github.com/spf13/cobra.(*Command).ExecuteContext(...)
        github.com/pydio/cells/vendor/github.com/spf13/cobra/command.go:888
github.com/pydio/cells/cmd.Execute()
        github.com/pydio/cells/cmd/root.go:432 +0xc8
main.main()
        github.com/pydio/cells/main.go:116 +0x68
```

**Comment by bsinou** [comment]
Hello,

Would you please so kind and rather use the forum for such questions? (With a bit more details if you want to raise your chances to get a useful reply).

Thanks,

**Comment by sergeycherepanov** [comment]
@bsinou thank, I will try. But currently in looks like an error in the migration script.

**Comment by cdujeu** [comment]
Hi @sergeycherepanov , should be fixed in v3.0.2
